app/services/irs_attempts_api/tracker.rb

```diff
@@ -16,7 +16,7 @@ def initialize(session_id:, request:, user:, sp:, cookie_device_uuid:,
     end
 
     def track_event(event_type, metadata = {})
-      return if !enabled?
+      return unless enabled?
```

Comment:
personally I want to ban `unless`

Reply:
I tend to lean the other way, but don't feel strongly<sup>1</sup>.

This left me curious: I find 106 matches for `unless`, and 104 for `if !` in Ruby files in the project. Seems like there's not a clear standard one way or the other.

I'll remove this one since this was a "might as well change this while I'm here" thing.

<sup>1</sup> When I was first learning Rails, I ended up writing `unless ... else` somewhere. A coworker told me that, though it may be valid Ruby, "people _will_ hurt you."